components/TextControls/src/BaseTextAreas/private/MDCBaseTextAreaTextView.m

```diff
@@ -0,0 +1,71 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "MDCBaseTextAreaTextView.h"
+#import "MDCTextControl.h"
+
+@implementation MDCBaseTextAreaTextView
+
+- (instancetype)init {
```

Comment:
Doesn't this just call `initWithFrame:CGRectZero`, I think this can be removed but I think we need to add `initWithCoder:`

Reply:
Yes, good call